### PR TITLE
mtmd: support dots3-note vision+audio

### PR DESCRIPTION
**Pull request by ngxson**
## Overview

Follow-up #27060 

Add MoE support for vision tower

Audio support is trivial, just good old whisper

Compare logits vs HF version:

  | test | tokens | NMSE | worst token cos |
  |---|---|---|---|
  | image 448x336 | 192 = HF | 1.6e-4 | 0.9982 |
  | audio 7.3s | 92 = HF | 1.6e-6 | 0.999999 |
  | audio 65s (chunked) | 750+63 = HF | 1.5e-6 | 0.999999 |

## Requirements

- I have read and agree with the [contributing guidelines](https://github.com/ggml-org/llama.cpp/blob/master/CONTRIBUTING.md)
- AI usage disclosure: yes

### CODE REVIEW

File: gguf-py/gguf/tensor_mapping.py [hunk 1]
```diff
@@ -2294,9 +2342,12 @@ class TensorNameMap:
             "encoder.layers.{bid}.fc1", # mimo-audio-tokenizer
         ),
 
-        MODEL_TENSOR.A_ENC_FFN_GATE: (),
+        MODEL_TENSOR.A_ENC_FFN_GATE: (
+            "audio_encoder.dots_encoder.speech_encoder.layers.{bid}.fc1_gate", # dots3note (split from fc1 in conversion code)
+),
```
Comment (CISC) [inline gguf-py/gguf/tensor_mapping.py]:
```suggestion
        ),
```

**Review by CISC — APPROVED** [review]
